src/main/java/org/jboss/aerogear/keycloak/metrics/ResourceExtractor.java

```diff
@@ -65,21 +65,23 @@ static String getResource(UriInfo uriInfo) {
     static String getURI(UriInfo uriInfo) {
         if (URI_METRICS_ENABLED) {
             List<String> matchedURIs = uriInfo.getMatchedURIs();
-            StringBuilder sb = new StringBuilder();
+            if (!matchedURIs.isEmpty()) {
```

Comment:
This makes sense. So previously we would try to access `matchedURIs.get(0)` even if there were no matched URIs.